feature/experimental/bgp/ate_tests/base_bgp_session_parameters/base_bgp_session_parameters_test.go

```diff
@@ -151,6 +149,19 @@ func verifyBGPCapabilities(t *testing.T, dut *ondatra.DUTDevice) {
 
 }
 
+// verifyAuthPassword checks that the dut applied configured auth password to bgp neighbors
```

Comment:
Remove since we're not supposed to be able to fetch the password back from the device.

Reply:
Removed this part of code, thank you